README.md

```diff
@@ -24,10 +24,24 @@ The process is divided in five steps:
 2. start by editing your colorscheme's information,
 3. define your colors,
 4. define your highlight groups and links,
-5. and generate your colorscheme.
+5. and generate your colorscheme (refer next step below).
```

Comment:
Note that the next paragraph says:

> Steps 2 to 5 are thoroughly described in the colorscheme template itself

Where there is:

    # Step 5: generation
    #
    # From a separate shell:
    #
    #     $ erb -T - bar.erb > bar.vim
    #
    # From Vim:
    #
    #     :!erb -T - % > %<.vim
    #
    # If this template comes with a Makefile, you can do it from a separate shell,
    # with the make program:
    #
    #     $ make

which, I think, covers the basics.

Reply:
ouh yeah, I didn't notice this one. probably my bad. but in the new changes I made the link to refer it to particular line to refer how to generate